BigBatch/Views/BatchedVC1.swift

```diff
@@ -408,6 +358,97 @@ class BatchedVC1: UIViewController, UITableViewDelegate, UITableViewDataSource,
+                                                                   ingredient12Amount: totalMlsArray[11],
+                                                                   ingredient13Amount: totalMlsArray[12],
+                                                                   ingredient14Amount: totalMlsArray[13],
+                                                                   ingredient15Amount: totalMlsArray[14],
+                                                                   ingredient16Amount: totalMlsArray[15],
+                                                                   singleIngredientAmount1: mlAmount1,
+                                                                   singleIngredientAmount2: mlAmount2,
+                                                                   singleIngredientAmount3: mlAmount3,
+                                                                   singleIngredientAmount4: mlAmount4,
+                                                                   singleIngredientAmount5: mlAmount5,
+                                                                   singleIngredientAmount6: mlAmount6,
+                                                                   singleIngredientAmount7: mlAmount7,
+                                                                   singleIngredientAmount8: mlAmount8,
+                                                                   singleIngredientAmount9: mlAmount9,
+                                                                   singleIngredientAmount10: mlAmount10,
+                                                                   singleIngredientAmount11: mlAmount11,
+                                                                   singleIngredientAmount12: mlAmount12,
+                                                                   singleIngredientAmount13: mlAmount13,
+                                                                   singleIngredientAmount14: mlAmount14,
+                                                                   singleIngredientAmount15: mlAmount15,
+                                                                   singleIngredientAmount16: mlAmount16,
+                                                                   singleDilutionAmount: singleCoctailDilutionMls,
+                                                                   dilutionAmount: mlDilutionAmouunt,
+                                                                   dilutionName: batchedDilutionType,
+                                                                   totalVolume: batchedTotalVolume,
+                                                                   singleCocktailAmount: singleCocktailVolume,
+                                                                   cocktailCount: batchedInitialCocktailsNumber,
+                                                                   cocktailName: batchedCocktailName), animated: true)
+        
```

Comment:
Passing this many parameters into a viewController works.. but we should really be passing in a single model object here instead